src/redisenterprise/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+0.1.5
++++++++++
+- Added support for new enterprise SKU E1
```

Comment:
To upgrade version, not only history notes need to be updated. Also find the version defined in `setup.py` or `version.py`

Reply:
updated the setup.py. Seems like We don't have version.py in our module